rails.md

```diff
@@ -5,29 +5,28 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. MVC (Model View Controller) is a pattern for the architecture of a software program. Give a brief description of each component and describe how Ruby on Rails handles MVC.
 
-  Your answer:
-
-  Researched answer:
-
-
+  Your answer: 
+  Model = contains the structural base code before it is told what to do.
+  view = what the user sees
+  controller = contains all the methods and tells the model what to do.
 
 2. Using the information given, fill in the blanks to complete the steps for creating a new view in a Rails application.
 
-  Step 1: Create the _________ in the file config/routes.rb
+  Step 1: Create the __route_______ in the file config/routes.rb
   ```
   get '/about' => 'statics#about'
   ```
 
-  Step 2: Create the ____________ in the file _____________
+  Step 2: Create the ___controller_________ in the file ____app/controllers/statics_controllers.rb _________
   ```
-  class ____________ < ApplicationController
-    def ___________
-      render ________________
+  class ____StaticsController________ < ApplicationController
+    def ____index_______
+      render _json: index?_______________
```

Comment:
You could either render the view at "index.html.erb" or render a list of from the database "render json: Model.all"